cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowUtil.qll

```diff
@@ -43,7 +43,7 @@ class Node extends TNode {
   /**
    * INTERNAL: Do not use. Alternative name for `getFunction`.
    */
-  Function getEnclosingCallable() { result = this.getFunction() }
+  final Function getEnclosingCallable() { result = unique( | | this.getFunction()) }
```

Comment:
I guess without monotonic aggregates active this does the right thing, but perhaps we should still write this as `unique(Function f | f = this.getFunction() | f)` to make it correct in both versions of aggregate semantics?  It is annoying that we can't simply write `unique(this.getFunction())` as this would be short-hand for the right thing, namely `unique(Function f | f = this.getFunction() | f)`.

Reply:
Done.